src/main/java/uk/gov/hmcts/cmc/claimstore/events/ccd/CCDCaseHandler.java

```diff
@@ -121,7 +121,7 @@ public void requestMoreTimeForResponse(CCDMoreTimeRequestedEvent event) {
         }
     }
 
```

Comment:
no point changing this in transactional as calling service method is not transactional

Reply:
It is transactional in `CountyCourtJudgmentService`. So let it be `@TransactionalEventListener`

Reply:
left it as @TransactionalEventListener. Tested in local and was able to see CCJ is posted to CCD